iterations_runners/iterations_runner.java

```diff
@@ -131,7 +131,10 @@ public void getInstJSON(StringBuilder sb, int iterNum) {
         System.loadLibrary("kruntime");
     }
 
+    static long UNINITIALISED_TSR = -1;  // All ones
```

Comment:
Again, if we're emulating unsigned numbers (as a comment below suggests), `-1` doesn't seem like the best choice.